docs/contributor/arch/027-switch-helm-packaging-from-kustomize.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Switch Kustomize Packaging to Helm
+status: Accepted
+date: 2025-08-21
+---
+
+# 26. Switch Kustomize Packaging to Helm
+
+## Context
+
+For deploying the module across multiple environments, templating capabilities are frequently required. While Kustomize was initially chosen due to its integration with kubebuilder (which generates artifacts using Kustomize), its role in our setup has become minimal. At this point, Kustomize is only used for CRD generation.
+
+Since these CRDs are unaffected by the deployment layer, we now have the opportunity to switch the main deployment mechanism to Helm, which is the de-facto standard for Kubernetes packaging and offers significantly more powerful templating and configuration features. The proposal is therefore to convert the existing `config/` folder into a Helm chart, and adapt the deployment and testing tooling accordingly.
```

Comment:
```suggestion
Because these CRDs are unaffected by the deployment layer, we now have the opportunity to switch the main deployment mechanism to Helm, which is the de-facto standard for Kubernetes packaging and offers significantly more powerful templating and configuration features. Therefore, the proposal is to convert the existing `config/` folder into a Helm chart, and adapt the deployment and testing tooling accordingly.
```